src/wmtk/MultiMeshManager.cpp

```diff
@@ -344,8 +368,299 @@ std::string MultiMeshManager::parent_to_child_map_attribute_name(long index)
 {
     return fmt::format("map_to_child_{}", index);
 }
+std::array<attribute::MutableAccessor<long>, 2> MultiMeshManager::get_map_accessors(
+    Mesh& my_mesh,
+    ChildData& c)
+{
+    Mesh& child_mesh = *c.mesh;
+    const auto& child_to_parent_handle = child_mesh.m_multi_mesh_manager.map_to_parent_handle;
+    const auto& parent_to_child_handle = c.map_handle;
+
+
+    return std::array<attribute::MutableAccessor<long>, 2>{
+        {my_mesh.create_accessor(parent_to_child_handle),
+         child_mesh.create_accessor(child_to_parent_handle)}};
+}
+std::array<attribute::ConstAccessor<long>, 2> MultiMeshManager::get_map_const_accessors(
+    const Mesh& my_mesh,
+    const ChildData& c) const
+{
+    const Mesh& child_mesh = *c.mesh;
+    const auto& child_to_parent_handle = child_mesh.m_multi_mesh_manager.map_to_parent_handle;
+    const auto& parent_to_child_handle = c.map_handle;
+
+
+    return std::array<attribute::ConstAccessor<long>, 2>{
+        {my_mesh.create_const_accessor(parent_to_child_handle),
+         child_mesh.create_const_accessor(child_to_parent_handle)}};
+}
 std::string MultiMeshManager::child_to_parent_map_attribute_name()
 {
     return "map_to_parent";
 }
+
+void MultiMeshManager::update_map_tuple_hashes(
+    Mesh& my_mesh,
+    PrimitiveType primitive_type,
+    const std::vector<std::tuple<long, std::vector<Tuple>>>& simplices_to_update,
+    const std::vector<std::tuple<long, std::array<long, 2>>>& split_cell_maps)
+{
+    // spdlog::info(
```

Comment:
I guess that's debug stuff that can be deleted.

Reply:
yep. next PR as i mention elsewhere